docs/related.md

```diff
@@ -70,7 +70,7 @@ Tools layered on top of Helm or Tiller.
 - [Chartify](https://github.com/appscode/chartify) - Generate Helm charts from existing Kubernetes resources.
 - [Codefresh](https://codefresh.io) - Kubernetes native CI/CD and management platform with UI dashboards for managing Helm charts and releases
 - [Cog](https://github.com/ohaiwalt/cog-helm) - Helm chart to deploy Cog on Kubernetes
-- [Drone.io Helm Plugin](http://plugins.drone.io/ipedrazas/drone-helm/) - Run Helm inside of the Drone CI/CD system
+- [Drone.io Helm Plugin](https://plugins.drone.io/ipedrazas/drone-helm/) - Run Helm inside of the Drone CI/CD system
```

Comment:
Another link that does not appear to support `https`.  Can you please revert to `http`